sunkit_image/offlimb_enhance.py

```diff
@@ -295,3 +295,181 @@ def normalizing_radial_gradient_filter(
         ) / radial_intensity_distribution_summary[i]
 
     return sunpy.map.Map(data, smap.meta)
+
+
+def fourier_normalizing_radial_gradient_filter(
```

Comment:
Is this added the `__all__`? (If there is not one, it should be added).

Reply:
Done